plugins/modules/rds_cluster.py

```diff
@@ -1019,6 +1027,12 @@ def changing_cluster_options(modify_params, current_cluster):
         if apply_immediately is not None:
             changing_params["ApplyImmediately"] = apply_immediately
 
```

Comment:
I guess we should also check some of the limitations for the implementation: see section `Limitations for stopping and starting Aurora DB clusters` https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-cluster-stop-start.html